tests/conftest.py

```
@@ -1,32 +1,114 @@
+import logging
 import os
-from typing import Any, Generator
+import tempfile
+import time
+from dataclasses import dataclass
+from typing import Any, Dict, Generator
+from unittest.mock import Mock
 
+import boto3
 import pytest
+from google.genai import types
```

Pylint (3.3.7)

[error] No name 'genai' in module 'google'

(E0611)